.gitmodules

```diff
@@ -101,11 +101,14 @@
 	path = software/example-pwm
 	url = https://github.com/sifive/example-pwm.git
 [submodule "software/clic-nested-interrupts"]
-        path = software/clic-nested-interrupts
-        url = https://github.com/sifive/example-clic-nested-interrupts.git 
+	path = software/clic-nested-interrupts
+	url = https://github.com/sifive/example-clic-nested-interrupts.git
 [submodule "software/mem-latency"]
 	path = software/mem-latency
 	url = https://github.com/sifive/benchmark-mem-latency.git
+[submodule "software/example-hwperfmon"]
```

Comment:
Better to call "hardware-performance-monitor", without the "example"

Reply:
fixed.